src/main/java/com/epam/eighty/repository/TopicRepository.java

```diff
@@ -10,4 +12,6 @@
 @Repository("topicRepo")
```

Comment:
I think there's no need to mark interfaces with @Repository as u have already has
```xml
<neo4j:repositories base-package="com.epam.eighty.repository" />
```
in your config file which is used to enabled repository support. I'm not sure and this needs to be clarified but it's possible that repository beans is instantiated twice.
Plus, it's not a good idea to manually hardcode bean names.